Add docs for OpenID API proxy and general reorg of auth docs (#297)
kiali/kiali.io: content/documentation/staging/configuration/authentication/_index.adoc

---
title: "Authentication"
date: 2020-08-25T00:00:00+02:00
draft: false
weight: 9
hideIndex: true
---

Kiali supports four authentication mechanisms:

* link:{{< relref "anonymous" >}}[anonymous]: gives free access to Kiali.
* link:{{< relref "openid" >}}[openid]: requires authentication through a third-party service to access Kiali.
* link:{{< relref "openshift" >}}[openshift]: requires authentication through the OpenShift authentication to access Kiali.

> * link:{{< relref "token" >}}[token]: requires to provide a Kubernetes ServiceAccount token to ccess Kiali.

Reviewer: ```suggestion
* link:{{< relref "token" >}}[token]: requires user to provide a Kubernetes ServiceAccount token to access Kiali.
```

The default authentication mechanism for OpenShift clusters is `openshift`. For
all other Kubernetes clusters, the default mechanism is `token`.

> Read the dedicate page of each authentication mechanism (by clicking on the

Reviewer: ```suggestion
Read the dedicated page of each authentication mechanism (by clicking on the
```

> bullets) to learn more. All mechanisns, but `anonymous`, support link:{{<

Reviewer: ```suggestion
bullets) to learn more. All mechanisms other than `anonymous` support link:{{<
```
